packages/gatsby-plugin-manifest/README.md

```diff
@@ -50,6 +50,7 @@ module.exports = {
         // see https://developers.google.com/web/fundamentals/web-app-manifest/#display
         display: `standalone`,
         icon: `src/images/icon.png`, // This path is relative to the root of the site.
+        include_favicon: true, // Include favicon
```

Comment:
I think option keys ought to be camelCased for consistency
```suggestion
        includeFavicon: true, // Include favicon
```

Reply:
Hello @sidharthachatterjee Thank you. All other option keys are mentioned in snakeCase in documentation - https://www.gatsbyjs.org/packages/gatsby-plugin-manifest/?=manife#automatic-mode I would rather refactor all to camelCase or leave this one as is. Please suggest.

Reply:
I agree with @gurpreet-hanjra better to keep it consistent. Might be something we want to consider for v3